### ISSUE TITLE
mw -Y cant prompt for password

### ISSUE THREAD

**Issue by nabakolu** [LukeSmithxyz/mutt-wizard#675]
When calling `mw -Y` I do not get a prompt for my password.
However, when using pass, e.g. `pass show user@email` I **do** get a prompt.
Currently I have hacked it with remapping O like so:
`macro index O "<shell-escape>pass show lnabakowski@gmail.com; mw -Y<enter>"`
So I get prompted to use my password by pass, before `mw -Y` is called.
That currently works okay, but it feels a bit hacky. 
I don't know much about pass and gpg, so I havent really any clue on how to troubleshoot that.

**Comment by Kr1ss-XD** [comment]
Is [pam-gnupg](https://github.com/cruegge/pam-gnupg) installed ?

**Comment by nabakolu** [comment]
Nope, but afaik its an optional dependency

**Comment by LukeSmithxyz** [comment]
Yeah, while I advise that `pam-gnupg` be installed, this underlying problem has been an issue for a while. There is some way the command is called in the script that does not successfully evoke pinentry even if properly installed.

It'd be nice to figure this out for everyone.

**Comment by MarkusLohmayer** [comment]
not sure, but is it possible that the issue is that the mailsync script
https://github.com/LukeSmithxyz/mutt-wizard/blob/master/bin/mailsync#L31
is accessing the `$TTY` variable which is not exported.
If that is your problem, then you can either remove line 31 and put it in your `zshrc` or what have you .. or you can replace `$TTY` with `$(tty)`. Probably the former option makes more sense, assuming that GPG is potentially used by other things than `mailsync`.

I am not sure though if this helps anything if `mailsync` is called via `cron`.

on a tangent:
Actually what I would like to have is that on login, I am asked for the passphrase once and then done.
This is how SSH works for me.
But I could not find a way to set it up in this way.

**Comment by LukeSmithxyz** [comment]
Okay. I'll do some testing with the `$TTY`. That might be it.

And yeah, password-on-first-run is how the wizard *should* work by default.

**Comment by LukeSmithxyz** [comment]
Now that I think about it, I don't know if we've ever had cronjob compatibility *without* `pam-gnupg`. I guess that isn't stated in the README now because I removed it when I had temporarily removed the cronjob ability.

Now as to @nabakolu 's original issue. I can't replicate it here. I've disabled `pam-gnupg` and the `o` and `O` bindings in neomutt work fine and do indeed prompt me with a pinentry-gtk input box without further edits. Are there any potential quirks on your machine/configuration? Are you using an up-to-date version of mutt-wizard?

**Comment by nabakolu** [comment]
I did just install mutt-wizard from the aur. I have a fairly plain arch install, nothing really quirky, just bash and st.

**Comment by converge** [comment]
I fixed it in this way for me:
https://github.com/LukeSmithxyz/mutt-wizard/pull/687

I just did a quick and dirty fix on my side on my `~/.config/mutt/accounts/my-account.muttrc` to be like:
```bash
 macro index o "<shell-escape>mbsync -a<enter>" "run mbsync to sync example@account.com"
```
